client/reader/stream/reader-sidebar-organizations/index.jsx

```diff
@@ -25,12 +25,15 @@ export class ReaderSidebarOrganizations extends Component {
 	}
 
 	renderItems() {
-		const { path } = this.props;
+		const { path, translate } = this.props;
 		const organization = this.fetchOrganization();
+		const organizationSlug = organization.slug === 'a8c' ? translate( 'a8c' ) : translate( 'p2' );
+
 		return (
 			<>
 				<h2>
-					Following ({ organization.slug }) <a href="/following/mark-all">Mark all as seen</a>
+					{ translate( 'Following' ) } ({ organizationSlug }){ ' ' }
```

Comment:
When you have a phrase like this, it's important to translate the whole phrase, using a placeholder fo the part that's dynamic, since the order of subject/verb may change depending on the language.

See https://github.com/Automattic/wp-calypso/tree/trunk/packages/i18n-calypso#string-substitution

I'd expect it to look something like

```
translate( 'Following %s', { args: organizationSlug } )